tests/phpunit/tests/date/wpDate.php

```diff
@@ -61,4 +61,55 @@ public function test_should_keep_localized_slashes() {
 
 		$this->assertSame( $string, wp_date( 'F', $datetime->getTimestamp(), $utc ) );
 	}
+
+	/**
+	 * Test if no timestamp provides
+	 *
+	 * @ticket 53485
+	 */
+	public function test_no_timestamp() {
+
+		$this->assertSame( (string) strtotime( 'now' ), wp_date( 'U' ) );
```

Comment:
`time()` for current timestamp? `strtotime()` is a mess. :)

Reply:
done